src/QsCompiler/LlvmBindings/Interop/LLVMExecutionEngineRef.cs

```diff
@@ -0,0 +1,162 @@
+// Copyright (c) .NET Foundation and Contributors. All Rights Reserved. Licensed under the MIT License (MIT). See License.md in the repository root for more information.
```

Comment:
(nit) FYI: Some of the files have the fragment "See License.md in the repository root for more information." in the copyright line, some others not.

Reply:
Oops! Good catch, I'll make sure to remove that.

Reply:
Fixed in https://github.com/microsoft/qsharp-compiler/pull/1331, should be merged before this one to update feature branch.